chromium_src/chrome/browser/renderer_context_menu/render_view_context_menu.cc

```diff
@@ -78,11 +95,23 @@ void RenderViewContextMenu::RegisterMenuShownCallbackForTesting(
 BraveRenderViewContextMenu::BraveRenderViewContextMenu(
     content::RenderFrameHost* render_frame_host,
     const content::ContextMenuParams& params)
-  : RenderViewContextMenu_Chromium(render_frame_host, params) {
-}
-
+    : RenderViewContextMenu_Chromium(render_frame_host, params)
+#if BUILDFLAG(IPFS_ENABLED)
+    , ipfs_submenu_model_(this)
+#endif
+    {}
+#if BUILDFLAG(IPFS_ENABLED)
```

Comment:
This seems to be in a wrong place? Should go 2 lines below around the newly added switch cases?